src/main.ts

```diff
@@ -25,15 +25,15 @@ function createWindow() {
       preload: path.join(__dirname, 'preload.js'),
       webSecurity: false,
       allowRunningInsecureContent: true,
-      autoplayPolicy: "no-user-gesture-required"
+      autoplayPolicy: 'no-user-gesture-required',
     },
     autoHideMenuBar: true,
     kiosk: true,
     alwaysOnTop: true,
     // show: false, // turn on for .ge
```

Comment:
Will something like this work or this just should be removed?
```suggestion
    show: location.hostname.endsWith('.ge'),
```